src/cmd/singularity/cli/push.go

```diff
@@ -20,7 +20,7 @@ var (
 func init() {
 	PushCmd.Flags().SetInterspersed(false)
 
-	PushCmd.Flags().StringVar(&PushLibraryURI, "library", "https://library.sylabs.io", "")
+	PushCmd.Flags().StringVar(&PushLibraryURI, "library", "https://library.sylabs.io", "library to push to")
```

Comment:
For consistency with `singularity pull` this should probably be `"the library to push to"`

Reply:
Got it!